state/logs.go

```diff
@@ -66,6 +66,9 @@ var logIndexes = [][]string{
 	// the same time have a consistent ordering.
 	{"e", "t", "_id"},
 	{"e", "n"},
+
+	// This index is used for pruning logs by model and time.
+	{"e", "t"},
```

Comment:
I've backed out the logs-related changes. I couldn't reproduce my results from yesterday, which makes me think I was looking at the query planner results wrong the first time. The log pruning is slow (as shown in a link on the bug), but if it's using the {e, t, _id} index, adding an {e, t} index won't help. We may need to be smarter about bucketing time-spans, and dropping whole buckets at a time.